<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDlg</class>
 <widget class="BaseDlg" name="AddUserDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddUserDlg</string>
  </property>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>369</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>369</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddPageLoginNameLab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>92</y>
     <width>251</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>登录名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPagePasswordEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>299</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPageUserNameEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>127</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="ErrTipLab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>359</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
color:#df173e;</string>
   </property>
   <property name="text">
    <string>错误提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPageLoginNameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>127</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="AddPagePermissionLab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>178</y>
     <width>251</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>用户权限</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddPagePasswordLab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>264</y>
     <width>251</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AddPagePermissionCombo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>213</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="AddPageUserNameLab">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>92</y>
     <width>251</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPagePasswordSureEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>299</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="AddPagePasswordSureLab">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>264</y>
     <width>251</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
